rules/libyang.mk

```diff
@@ -1,23 +1,31 @@
 # libyang
 
-LIBYANG_VERSION_BASE = 0.16
-LIBYANG_VERSION = $(LIBYANG_VERSION_BASE).105
+LIBYANG_VERSION_BASE = 1.0
+LIBYANG_VERSION = $(LIBYANG_VERSION_BASE).70
 LIBYANG_SUBVERSION = 1
 
 export LIBYANG_VERSION_BASE
 export LIBYANG_VERSION
 export LIBYANG_SUBVERSION
 
-LIBYANG = libyang$(LIBYANG_VERSION_BASE)_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
+#LIBYANG = libyang$(LIBYANG_VERSION_BASE)_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
+LIBYANG = libyang_$(LIBYANG_VERSION)_$(CONFIGURED_ARCH).deb
 $(LIBYANG)_SRC_PATH = $(SRC_PATH)/libyang
 $(LIBYANG)_DEPENDS += $(SWIG_BASE) $(SWIG)
 SONIC_MAKE_DEBS += $(LIBYANG)
 SONIC_STRETCH_DEBS += $(LIBYANG)
 
-LIBYANG_DEV = libyang-dev_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
-$(eval $(call add_derived_package,$(LIBYANG),$(LIBYANG_DEV)))
+#LIBYANG_DEV = libyang-dev_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
+LIBYANG_DEV = libyang-dev_$(LIBYANG_VERSION)_$(CONFIGURED_ARCH).deb
```

Comment:
same question. Reason for changing the package name?

Reply:
Use the Debian package names built from libyang GitHub